transform/mattermost-analytics/models/marts/product/licenses/_product__licenses__models.yml

```diff
@@ -91,3 +91,55 @@ models:
         description: |
           Whether the expiration date is consistent in case there's multiple sources that report the license. True if 
           more than one expiration dates are reported, false else.
+
+  - name: dim_self_hosted_customers
```

Comment:
remove?

Reply:
I will drop it, I'm still deciding if we need this